roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -138,6 +138,10 @@ cpu_mode = host-passthrough
 {% else %}
 cpu_mode = host-model
 {% endif -%}
+live_migration_downtime = 500
+live_migration_downtime_steps = 10
+live_migration_downtime_delay = 75
+live_migration_uri = "qemu+ssh://%s/system?keyfile={{ nova.state_path }}/.ssh/id_rsa"
```

Comment:
libvirt connects be default as the root user, so if we use these keys for root, we're good.